### ISSUE TITLE
FileNotFoundError not being excepted on Windows

### ISSUE THREAD

**Issue by Honno** [Honno/rngtest#3]
```
_____________________ ERROR collecting tests/test_prng.py _____________________
tests\test_prng.py:4: in <module>
    from prng.cli import main
.tox\py36\lib\site-packages\prng\cli.py:7: in <module>
    from prng.store import load as load_
.tox\py36\lib\site-packages\prng\store.py:26: in <module>
    Path.mkdir(data_dir)
c:\python36\lib\pathlib.py:1248: in mkdir
    self._accessor.mkdir(self, mode)
c:\python36\lib\pathlib.py:387: in wrapped
    return strfunc(str(pathobj), *args)
E   FileNotFoundError: [WinError 3] The system cannot find the path specified: 'C:\\Users\\appveyor\\AppData\\Local\\prng\\prng'
```

**Comment by Honno** [comment]
Fixed in commit 39eecc5ecadb9973b1f4ecc02fafd376d9caf4b1

**Comment by Honno** [comment]
Fixed in commit 39eecc5ecadb9973b1f4ecc02fafd376d9caf4b1
